common/culture/pillars/wc_heritage.txt

```diff
@@ -52,7 +52,6 @@ heritage_jinyu = {
 	# If change, update heritage_gives_sea_raiding_trigger
 	parameters = {
 		can_travel_along_rivers = yes
-		culture_can_raid_at_sea_even_if_feudal = yes
```

Comment:
Why it's removed?

Reply:
It was a mistake and redundant. The parameter should only be in the tradition file. There is parameter `culture_can_raid_at_sea_even_if_feudal` and `culture_can_raid_at_sea_even_if_feudal_trigger`.

The cultures with `heritage_gives_sea_raiding_trigger` can already raid at sea, and if they pick `tradition_practiced_pirates` they will be able to raid while feudal due to `can_raid_trigger`

Reply:
It's localization thing too. If you read the tradition description, it says they are able to sea raid.

Reply:
Got it. It's been reverted.